config/locales/en.yml

```diff
@@ -386,7 +386,7 @@ en:
       maps: "Maps"
       no_results_found: "No results found; try searching for '<b>%{tag}</b>'"
       try_advanced_search: "Or try an <a href='%{url1}'>advanced search</a>"
-      contributors: "Contributors"
```

Comment:
Actually, keep the contributors too as we are using tag.show.contributors in line 40

Reply:
Done!